README.md

```diff
@@ -1 +1 @@
-# haxxor-news-clone
\ No newline at end of file
+Hacker News clone where users can post articles.
```

Comment:
I would also add information here about how to set up the app locally (like what versions of Ruby you need installed, the commands to set up the database, how to actually run it, etc.)